### ISSUE TITLE
[UX][mobile] Trusted publishing: missing menu item

### ISSUE THREAD

**Issue by webknjaz** [pypi/warehouse#13578]
**Describe the bug**

There's no way to navigate to the list of configured trusted connections on mobile. It is possible to navigate to a similar page under specific projects but I wanted to configure a pending trusted publisher and couldn't navigate there.

It is possible to open the page if you remember the link — https://test.pypi.org/manage/account/publishing/.
The menu item also becomes visible when I use the "pretend to be a desktop" option in Chrome Mobile.

**Expected behavior**

The trusted publishing menu item must be always visible.

**To Reproduce**

1. Open PyPI account settings or own projects list on mobile.
2. (not sure if this is needed) Possibly make your system fonts large.
3. See `Your Projects` and `Account Settings` in the menu but nothing OIDC-related.

**My Platform**

OnePlus 9R, Android, Chrome

**Additional context**

![Screenshot_2023-05-05-00-45-56-09_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/578543/236345491-e7bcec91-d54a-48f8-b80e-7cdeb882c2ea.jpg)
